superset/config.py

```diff
@@ -576,31 +578,44 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
 # Setup image size default is (300, 200, True)
 # IMG_SIZE = (300, 200, True)
 
-# Default cache timeout, applies to all cache backends unless specifically overridden in
+
+# Default cache config, applies to all cache backends unless specifically overridden in
 # each cache config.
-CACHE_DEFAULT_TIMEOUT = int(timedelta(days=1).total_seconds())
+def DEFAULT_CACHE_CONFIG_FUNC(  # pylint: disable=invalid-name
```

Comment:
I'm not sure we need a whole function for this. How about simply setting `FILTER_STATE_CACHE_CONFIG` and `EXPLORE_FORM_DATA_CACHE_CONFIG` to `NullCache` and override it to `SimpleCache` or raise exception when app is starting (depending on whether `app.debug` is true)?

Reply:
I agree, that's probably a better solution - I'll move the check for debug to `CacheManager.init_app`, seems like a cleaner approach 👍